packages/ui-server/utils/questioner.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
```

Comment:
@rishabh3112 why we need to define this as a `esModule`?

Reply:
Same here